source/ui/CLS/CLSButtonBar.cpp

```diff
@@ -81,23 +81,31 @@ void CLSButtonBar::removeButton(QAbstractButton *button)
 	}
 }
 
+void CLSButtonBar::removeButtons()
```

Comment:
this function is alike the `clearButtons()` except the deleting the button instance.

I think they are doing the same thing :question: